runtime_local/src/run-vehicle-app.py

```diff
@@ -31,20 +30,20 @@ def run_app(
     app_port: Optional[str] = None,
 ):
     program_args = [executable_path, *args]
-    envs = dict()
-    for service in get_services():
-        service_id = service.id
-        envs[get_dapr_app_id(service_id)] = service_id
 
-    if get_middleware_type() == MiddlewareType.DAPR:
+    if get_middleware_type() == MiddlewareType.NATIVE:
+        envs = dict()
+        envs["SDV_MIDDLEWARE_TYPE"] = "native"
```

Comment:
don't we also need to set the addresses of the other services?

Reply:
i also only know this from the readme: https://github.com/eclipse-velocitas/vehicle-app-python-sdk/blob/963924857751fba104579e16b8162f08beb8c31b/examples/seat-adjuster/README.md?plain=1#L23

Reply:
Yeah, somehow we still need the `get_serivces()` call which was removed.

Reply:
Basically yes, question is: Where to get it from? The config values within runtime.json aren't applicable at the moment. E.g. mqtt-broker is defining two ports. Which one to take?

Reply:
Used similar approach as for Kanto, for now